### PR TITLE
Design maintenance Jul 7 2023

### PR DESCRIPTION
**Pull request by micheal-parks**
Done in a session with nat. Mostly involves notification and modal tweaks.

### CODE REVIEW

**Comment by DTCurrie** [comment]
What was the fix for the notify "art" bug?

**Comment by micheal-parks** [comment]
@DTCurrie 

We restructured slots to be more intuitive, but this will actually require a follow up in app. There's an example added that creates a notification that would exist on the history page without that visual bug 🎉

File: src/stories/notify.stories.mdx [hunk 1]
```diff
@@ -178,11 +178,37 @@ Notify message of type info, success, warning, or error
     {() => `
 <v-notify
   variant='info'
-  title='This is the title.'
-  message='This is the message.'
+  title='You have an incoming life update'
+  message='LifeOS has been upgraded to a new version that includes great new things such as friends and family and you will not be alone anymore.'
```
Comment (mrloureed) [inline src/stories/notify.stories.mdx:182]: 🤣

**Review by mrloureed — APPROVED** [review]
LGTM!

**Pull request merged**